mods/doors/init.lua

```diff
@@ -505,6 +505,21 @@ doors.register("door_obsidian_glass", {
 		},
 })
 
+doors.register("door_bar", {
+		tiles = {{name = "doors_door_bar.png", backface_culling = true}},
+		description = "Steel Bar Door",
+		inventory_image = "doors_item_bar.png",
+		groups = {cracky = 1, level = 2},
+		sounds = default.node_sound_metal_defaults(),
+		sound_open = "doors_steel_door_open",
+		sound_close = "doors_steel_door_close",
+		recipe = {
+			{"xpanes:bar_flat", "xpanes:bar_flat"},
+			{"xpanes:bar_flat", "xpanes:bar_flat"},
+			{"xpanes:bar_flat", "xpanes:bar_flat"},
+		},
```

Comment:
12 lines above: only 1 indent needed.

Reply:
I already complaint about this. But elsewhere in this file, the indentation is the same.